pkg/core/app/app.go

```diff
@@ -117,6 +119,16 @@ func (a *App) SetupDocker() error {
 		a.logger.Warn(fmt.Sprintf("given docker network:(%v) is different from parsed docker network:(%v)", a.containerNetwork, net))
 	}
 
+	if a.kind == utils.DockerStart {
+		running, err := a.docker.IsContainerRunning(a.container)
```

Comment:
Even if the given container is different from the parsed container, we should provide the parsed container. i.e `cont` here. We are already logging a warning msg.

Reply:
corrected